Ginger/GingerCore/ALM/Qtest/ImportFromQtest.cs

```diff
@@ -219,151 +219,21 @@ public static BusinessFlow ConvertQtestTestSuiteToBF(QtestTestSuite testSuite)
-                                if (stepActivityVarOptionalVar == null)
-                                {
-                                    //no such variable value option so add it
-                                    stepActivityVarOptionalVar = new OptionalValue(paramSelectedValue);
-                                    ((VariableSelectionList)stepActivityVar).OptionalValuesList.Add(stepActivityVarOptionalVar);
-                                    if (isflowControlParam == true)
-                                    {
-                                        stepActivity.AutomationStatus = eActivityAutomationStatus.Development;//reset status because new param value was added
-                                    }
-                                }
-                                //set the selected value
-                                ((VariableSelectionList)stepActivityVar).SelectedValue = stepActivityVarOptionalVar.Value;
-                            }
-                            else
-                            {
-                                //try just to set the value
-                                try
-                                {
-                                    stepActivityVar.Value = paramSelectedValue;
-                                    if (stepActivityVar is VariableString)
-                                    {
-                                        ((VariableString)stepActivityVar).InitialStringValue = paramSelectedValue;
-                                    }
-                                }
-                                catch (Exception ex) { Reporter.ToLog(eLogLevel.ERROR, $"Method - {MethodBase.GetCurrentMethod().Name}, Error - {ex.Message}", ex); }
+                                stepActivity.Variables.Remove(stepActivityVar);
                             }
-
-                            //add linked variable if needed
-                            if (!string.IsNullOrEmpty(linkedVariable))
-                            {
-                                stepActivityVar.LinkedVariableName = linkedVariable;
-                            }
-                            else
-                            {
-                                stepActivityVar.LinkedVariableName = string.Empty; // clear old links
-                            }
-                        }
+                            stepActivityVar = new VariableSelectionList();
+                            stepActivityVar.Name = step.Params.Name;
+                            ObservableList<OptionalValue> optionalValuesList = new ObservableList<OptionalValue>();
+                            step.Params.Values.ForEach(z => optionalValuesList.Add(new OptionalValue(z)));
+                            ((VariableSelectionList)stepActivityVar).OptionalValuesList = optionalValuesList;
+                            ((VariableSelectionList)stepActivityVar).SetValue(step.Params.Value);
```

Comment:
Codacy found an issue: [Remove this unnecessary cast to 'VariableSelectionList'.](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=5794604)